sopel/modules/reddit.py

```diff
@@ -205,6 +205,23 @@ def comment_info(bot, trigger, match):
 
 def subreddit_info(bot, trigger, match, commanded=False):
     """Shows information about the given subreddit"""
+    match = match.lower()
```

Comment:
I'm not sure this'll play well with this line: `r.subreddits.search_by_name(match, exact=True)`

Maybe use a different name for that?